package.json

```diff
@@ -91,33 +91,33 @@
     "react": "18.2.0",
     "react-apollo": "~3.1.5",
     "react-dom": "18.2.0",
-    "react-hook-form": "^6.11.0",
-    "react-native": "0.71.8",
+    "react-hook-form": "^7.12.2",
```

Comment:
This looks like a major update. Have you read the changelog for this package? How much does it affect our code?

Reply:
also here, no changes effecting our code